src/chexus/validate.py

```diff
@@ -56,6 +57,8 @@ def format_details(self) -> str:
         return details
 
     def format_summary(self) -> str:
+        if self.skipped:
+            return f"{self.validator.name}: ?/{self.checks} (requires scipp)\n"
```

Comment:
You can't technically tell whether this skips because scipp is unavailable of for a different reason. Minor comment, though, I leave it up to you to decide what to do.

Reply:
I considered replacing the `self.skipped` boolean with an optional `self.skipped_reason` string. Then the function body would be replaced by
```python
if self.skipped_reason:
    return f"{self.validator.name}: ?/{self.checks} ({self.skipped_reason})\n"
```
with the advantage that we have to specify the reason for skipping in the same place where we mark the validator as skipped.

However, since we only skip validators for a single reason right now (scipp being unavaliable) that felt a bit overkill.

Reply:
I think the (length of the) whole discussion shows that the try/except approach and the `skipped` flag are causing more trouble than they have benefit. I suggest you simply select validators based on what libraries they require (as suggested earlier), then the whole discussion it moot.